### ISSUE TITLE
[FEATURE] Death Pits

### ISSUE THREAD

**Issue by Skitter1200** [eternalUnion/UltraPain#32]
[Please do not close the request after a revision release is posted. Request will be closed by the developer when it is implemented to the base version or when it is flagged as not planned]

# Feature
It's kind of weird that the bottomless pits that kill enemies only teleport V1 back with a slap on the wrist. That's a bit too merciful for a pain enthusiast like me.

# Usage
Any pitfall trap that teleports the player (including the ocean in Wrath) should kill you instead, because if you're dumb enough to fall into one of those then you deserve to perish.

# Extras
Maybe implement an option in memes for a funny sound effect when falling to your death?

**Comment by UntotenTheo** [comment]
Interesting idea, not sure if its possible to implement (I just reply, I don't actually code lol) but it would be an interesting addition.

In regards to the Extras section, could you provide an example of a funny sound effect? The only one that comes to mind for me is the Falling Metal Pipe sound.

**Comment by eternalUnion** [comment]
Yeah could do that, but wrath could be a problem when the tail throws the player in the ocean

**Comment by UntotenTheo** [comment]
A special exception could be made for that level, either by default or have it be configurable via the Leviathan section (or wherever you see fit, as I don't know if you plan on having a lot of Level/Hazard changes)

**Comment by eternalUnion** [comment]
https://github.com/eternalUnion/UltraPain/releases/tag/fr32-1.0.2-0
* Audio is DISABLED by DEFAULT
* This field is under Main Panel > Level Tweaks > Death by pit
* Audio file is located in the asset folder and can be changed (must be in .wav)
